.github/workflows/install.yaml

```diff
@@ -16,7 +16,7 @@ on:
 jobs:
   install:
     name: Install
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
Maybe we can utilize GitHub has linux/arm64 runner.

Ref: https://docs.github.com/en/actions/using-github-hosted-runners/using-github-hosted-runners/about-github-hosted-runners#standard-github-hosted-runners-for-public-repositories